### ISSUE TITLE
Model Outputs 

### ISSUE THREAD

**Issue by OrientKing** [JingyinLin/PKEcho-Net#3]
This is an excellent work, when reading the paper and the code, we observe that the model has 4 outputs of the same shape, how to deal with these outputs respectively during training and testing? Looking forward to your reply, thank you.

**Comment by JingyinLin** [comment]
During training, all outputs are used to calculate the loss function, supervising the predictor and each kernel refinement in the KSS module. Only the last output is used for evaluation when testing.
